### PR TITLE
Customer retrieve from db done logic between DAO and interface

### PR DESCRIPTION
**Pull request by LordStacker**
Customer retrieve from db done logic between DAO and interface, Mapping the UI need to be done (Me on weekend)

### CODE REVIEW

File: src/main/java/dk/easv/Main.java [hunk 1]
```diff
@@ -18,7 +18,7 @@ public class Main extends Application {
     @Override
     public void start(Stage stage) throws IOException {
         if (Files.exists(Config.CONFIG_PATH.getUrl())){
-            RootController controller = ControllerFactory.loadFxmlFile(ViewType.LOGIN);
+            RootController controller = ControllerFactory.loadFxmlFile(ViewType.USERS_VIEW);
```
Comment (patrikvalentiny) [inline src/main/java/dk/easv/Main.java:23]: do not push this please if you are not changing something else than this line

File: src/main/java/dk/easv/dal/dao/CustomerDAO.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    "select * from dbo.[customer]"
+            );
+            ResultSet rs = ps.executeQuery();
+            while (rs.next()) {
+                int dbId = rs.getInt("customer_id");
+                String dbName = rs.getString("name");
+                String dbEmail = rs.getString("email");
+                String dbAddress = rs.getString("address");
+                int dbZipcode = rs.getInt("zipcode");
+                Customer customer = new Customer(dbId,dbName,dbEmail,dbAddress,dbZipcode);
```
Comment (patrikvalentiny) [inline src/main/java/dk/easv/dal/dao/CustomerDAO.java]: last argument use city be

File: src/main/java/dk/easv/gui/controllers/WorkerViewController.java [hunk 3]
```diff
@@ -1,29 +1,50 @@
...
 import java.net.URL;
+import java.sql.SQLException;
 import java.util.ResourceBundle;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
 
-public class WorkerViewController implements Initializable {
+public class WorkerViewController extends RootController{
+
+    CRUDLogic crudLogic = new CRUDLogic();
```
Comment (patrikvalentiny) [inline src/main/java/dk/easv/gui/controllers/WorkerViewController.java]: model

File: src/main/java/dk/easv/gui/controllers/WorkerViewController.java [hunk 4]
```diff
@@ -1,29 +1,50 @@
...
+
+    ConcurrentMap<Integer, Customer> customers = new ConcurrentHashMap<>();
```
Comment (patrikvalentiny) [inline src/main/java/dk/easv/gui/controllers/WorkerViewController.java]: keep this in model

**Review by patrikvalentiny — CHANGES_REQUESTED** [review]

File: src/main/java/dk/easv/dal/dao/CustomerDAO.java (hunk 2 above)
Comment (LordStacker) [reply src/main/java/dk/easv/dal/dao/CustomerDAO.java]: Solved

**Review by patrikvalentiny — APPROVED** [review]

**Pull request merged**